install_script.sh.template

```diff
@@ -300,6 +300,17 @@ function remove_rpm_gpg_keys() {
     done
 }
 
+function get_relevant_gpg_keys() {
+    local minorVersion="$1"
+    shift
+    local gpgKeys=("$@")
+    # The last GPG key is only used on agent 7.35.X and lower
+    if [ -z "$minorVersion" ] || [ "$minorVersion" -gt 35 ]; then
+        unset 'gpgKeys[${#gpgKeys[@]}-1]'
```

Comment:
I fear this might bite us in the future, if we ever remove the key from the initial array.

Do you think we could process the array once, ideally immediately after defining it?

Reply:
🤦 I think my description was scuffed, I need to check but I think it's only popping from the local array `gpgKeys`

Reply:
Sure, but it would still stop installing/using the oldest key. It currently works fine because the oldest key is the one we want to remove for this version, but this won't hold if the list of keys change

Reply:
Ah yes but when we're doing the gpg key rotation in june we'll just have to add or something like that
```
if [ -z "$minorVersion" ] || [ "$minorVersion" -gt 52 ]; then
        unset 'gpgKeys[${#gpgKeys[@]}-2]'
```
inside of the same function no ?